process/block/baseProcess.go

```diff
@@ -670,6 +667,11 @@ func (bp *baseProcessor) removeBlockBodyOfHeader(headerHandler data.HeaderHandle
 		return err
 	}
 
+	err = bp.blockProcessor.removeStartOfEpochBlockDataFromPools(headerHandler, bodyHandler)
```

Comment:
Is this method actually needed? Shouldn't the rewardTxProcessor and validatorInfoPreprocessor remove this information on the 
`err := preproc.RemoveTxBlockFromPools(blockBody, tc.miniBlockPool)`

Reply:
Actually we don't have pre-processor for Rewards and Peer in meta, this is why this method appeared, just to clean also block data (txs and miniblocks) for meta node

Reply:
ok